app/util/conf/base_config_loader.py

```diff
@@ -105,6 +105,11 @@ def configure_defaults(self, conf):
         conf.set('https_cert_file', None)
         conf.set('https_key_file', None)
 
+        # Default values for pagination requests
```

Comment:
That makes sense, although it has not really been a problem in my experience. @wjdhollow you may want to file an issue if you feel strongly about it -- we'd want to do that all in one change instead of piecemeal.